command/pr.go

```diff
@@ -418,32 +418,39 @@ func printPrs(w io.Writer, totalCount int, prs ...api.PullRequest) {
 		reviews := pr.ReviewStatus()
 
 		if pr.State == "OPEN" {
-			if checks.Total > 0 || reviews.ChangesRequested || reviews.Approved {
+			reviewStatus := reviews.ChangesRequested || reviews.Approved || reviews.ReviewRequired
+			if checks.Total > 0 || reviewStatus {
```

Comment:
This fix prevents the `Review required` bit appearing on the _same_ line as the PR title when there are no checks. I've noticed this on your screenshots and got surprised because the same-line behavior was not intended. This ensures that checks & reviews _always_ appear on their own line below the PR title (until we decide otherwise).